### PR TITLE
[ADD] l10n_ar: RG 5003: 'A' documents and Monotributistas 

### PR DESCRIPTION
**Pull request by zaoral**
latam task 573 / ticket adhoc 40407
---

### Description of the issue/feature this PR addresses:

We have a new RG from AFIP No 5003 that said that a Responsable Inscriptio company can issue A documents to Monotributista AFIP Resposanbility Types, before this regulation we were only able to B and C documents.

### Current behavior before PR:

Not able to issue A documents to Monotributista AFIP Responsibility Types.

## Desired behavior after PR is merged:

Be able to issue  A documents to Monotributista AFIP Responsibility Types.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by jjscarafia — APPROVED** [review]
Technical review +1

**Comment by zaoral** [comment]
Hello @william-andre 
Please could you help us to review this PR?

Best Regards

cc @jco-odoo  @jjscarafia @vbe-odoo @ren-odoo @Bruno-Zanotti

File: addons/l10n_ar/views/report_invoice.xml [hunk 1]
```diff
@@ -239,6 +239,14 @@
 
             </div>
         </xpath>
+
+        <!-- RG 5003: Add legend for 'A' documents that have a Monotribuista receptor -->
+        <p name="comment" position="after">
+            <p t-if="o.partner_id.l10n_ar_afip_responsibility_type_id.code in ['6', '13'] and o.l10n_latam_document_type_id.l10n_ar_letter == 'A'" >
+                El crédito fiscal discriminado en el presente comprobante, sólo podrá ser computado a efectos del Régimen de Sostenimiento e Inclusión Fiscal para Pequeños Contribuyentes de la Ley Nº 27.618
+            </p>
+        </p>
```
Comment (william-andre) [inline addons/l10n_ar/views/report_invoice.xml:248]: Keep it in English and update the `pot`

**Review by william-andre — CHANGES_REQUESTED** [review]
Seems good. Would be great if I could have a functional validation, I have no idea where/how to find the docs with the information you provided. (JCO is not available this week)

File: addons/l10n_ar/views/report_invoice.xml (hunk 1 above)
Comment (zaoral) [reply addons/l10n_ar/views/report_invoice.xml:248]: Hi @william-andre
Thank you for the review.

> Seems good. Would be great if I could have a functional validation, I have no idea where/how to find the docs with the information you provided. (JCO is not available this week)

Indeed, @vbe-odoo gives us a functional review +1 on the Latam instance. Valentino, please could you give us functional +1 in this PR on Github just to have the record that has been functionally validated? We know Jos is not here this week that is the reason why we are asking you. We have a deadline for this RG so we need to merge it this week.

> Keep it in English and update the pot

Not sure what to do here. This paragraph only appears in local invoices and is an extract from an RG that as far as I know requires that exact text: that is the reason we leave it in Spanish directly and we do not add any English translation. What do you think about this @MarianaLuzTorres  @jjscarafia?

**Comment by vbe-odoo** [comment]
Hi @zaoral @william-andre 
This PR LGTM, and is ready for technical review. FUN tests were done and passed.
As @zaoral said, the text should be only in Spanish due to a legal requirement, and it's something that needs to be merged this week due to the fact this legal requirement starts to be mandatory next week.

Thanks!
cc' @ren-odoo

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**